docs/resources/workload.md

```diff
@@ -175,13 +185,22 @@ Optional:
 
 - **port** (Number) TCP Socket Port.
 
+<a id="nestedblock--container--readiness_probe--grpc"></a>
+
+### `container.readiness_probe.grpc`
+
+Optional:
+
+- **port** (Number) gRPC Port.
+
 <a id="nestedblock--container--volume"></a>
 
 ### `container.volume`
 
 Required:
 
 - **uri** (String) URI of volume at cloud provider.
+- **recovery_policy** (String) Only applicable to persistent volumes, this determines what Control Plane will do when creating a new workload replica if a corresponding volume exists. Available Values: `retain`, `recycle`. Default: `retain`.
```

Comment:
We can leave recovery_policy, but please add a deprecated warning. This field is no longer used.

Reply:
@MajidAbuRmila